examples/sph/run_show_all_sph_kernels.py

```
@@ -21,16 +21,16 @@
 
 def compute_integ_3d(q, W):
     if hasattr(np, "trapezoid"):
-        integrate_func = getattr(np, "trapezoid")
+        integrate_func = np.trapezoid
     else:
-        integrate_func = getattr(np, "trapz")
+        integrate_func = np.trapz  # noqa: NPY201
     return integrate_func(4 * np.pi * q**2 * W, q)
 
 
 def plot_test_sph_kernel(q, f, df, W, dW, title, ax):
```

Ruff (0.16.0)

[warning] Unused function argument: `f`

(ARG001)